frontend/src/v5/ui/routes/dashboard/projects/federations/federationSettingsForm/federationSettingsForm.component.tsx

```diff
@@ -0,0 +1,405 @@
+			isValid={formState.isValid}
+		>
+			<SectionTitle>Federation information</SectionTitle>
+			<ShareTextField
+				label="ID"
+				value={federation._id}
+				lightLabel
+			/>
+			<Controller
+				name="name"
+				control={control}
+				render={({ field }) => (
+					<TextField
+						{...field}
+						label={formatMessage({ id: 'federations.settings.form.name', defaultMessage: 'Name' })}
+						required
+						error={!!errors.name}
+						helperText={errors.name?.message}
+					/>
+				)}
+			/>
+			<Controller
+				name="desc"
+				control={control}
+				render={({ field }) => (
+					<TextField
+						{...field}
+						label={formatMessage({ id: 'federations.settings.form.description', defaultMessage: 'Description' })}
+						error={!!errors.desc}
+						helperText={errors.desc?.message}
+					/>
+				)}
+			/>
+			<FlexContainer>
+				<FormControl>
+					<InputLabel id="unit-label" required>
+						<FormattedMessage id="federations.settings.form.unit" defaultMessage="Units" />
+					</InputLabel>
+					<Select
+						labelId="unit-label"
+						defaultValue={defaultValues.unit}
+						{...register('unit')}
+					>
+						{Object.keys(UNITS).map((unit) => (
+							<MenuItem key={unit} value={unit}>
+								<FormattedMessage id={`federations.settings.form.unit.${unit}`} defaultMessage={UNITS[unit]} />
+							</MenuItem>
+						))}
+					</Select>
+				</FormControl>
+				<Controller
+					name="code"
+					control={control}
+					render={({ field }) => (
+						<TextField
+							{...field}
+							label={formatMessage({ id: 'federation.settings.form.code', defaultMessage: 'Code' })}
+							error={!!errors.code}
+							helperText={errors.code?.message}
+						/>
+					)}
+				/>
+			</FlexContainer>
+			<FormControl>
+				<InputLabel id="default-view-label">
+					<FormattedMessage id="federations.settings.form.view" defaultMessage="Default View" />
+				</InputLabel>
+				<SelectView
+					labelId="default-view-label"
+					defaultValue={defaultValues.defaultView}
+					{...register('defaultView')}
+				>
+					{[EMPTY_VIEW].concat(federation.views || []).map((view) => (
+						<MenuItemView
+							key={view._id}
+							value={view._id}
+						>
+							{view.hasThumbnail ? (
+								<Thumbnail
+									src={getThumbnail(view._id)}
+									alt={view.name}
+								/>
+							) : (
+								<ThumbnailPlaceholder />
+							)}
+							<ViewLabel>
+								{view.name}
+							</ViewLabel>
+						</MenuItemView>
+					))}
+				</SelectView>
+			</FormControl>
+			<SectionTitle>GIS servey point</SectionTitle>
+			<FlexContainer>
+				<Controller
+					name="latitude"
+					control={control}
+					render={({ field }) => (
+						<UnitTextField
+							{...field}
+							labelname={formatMessage({ id: 'federations.settings.form.lat', defaultMessage: 'LATITUDE' })}
+							labelunit={formatMessage({ id: 'federations.settings.form.lat.unit', defaultMessage: 'decimal' })}
```

Comment:
I am not sure how the UNITS array should be used in this context. Do you want me to create another array for latitude and longitude?

Reply:
I was thinking it for the labelUnit not for label name, to get the units array and index it by decimal. 
But now I'm thinking it might be better to leave this way, I can see that the casing here doesnt match the one in the array.